scripts/build.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
+const path = require('path');
+const generate = require('./generate');
+
+const THEME_DIR = path.join(__dirname, '..', 'theme');
+
+if (!fs.existsSync(THEME_DIR)) {
+    fs.mkdirSync(THEME_DIR);
+}
+
+module.exports = async () => {
+    const base = await generate();
+
+    return Promise.all([
+        fs.promises.writeFile(
+            path.join(THEME_DIR, 'onenord.json'),
```

Comment:
Better is to keep the same name `OneNord-color-theme.json`, because adding `-color-theme` at the end would show color palette, warnings/errors, etc. when the file is opened in VS Code.